### PR TITLE
AAP-8656 updated version numbers across document

### PR DESCRIPTION
**Pull request by emurtoug**
Per [AAP-8656](https://issues.redhat.com/browse/AAP-8656), updated the [Red Hat Ansible Automation Platform Installation Guide](https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html/red_hat_ansible_automation_platform_installation_guide/index) to remove version number artefacts where appropriate.

### CODE REVIEW

File: downstream/modules/platform/proc-installing-the-ansible-builder-rpm.adoc [hunk 1]
```diff
@@ -198,32 +198,32 @@ To upgrade between minor releases of AAP 2, use this general workflow.
...
 $ cd ..
 ----
 
-Copy the 2.1.2-1 inventory file into the 2.2.0-7 bundle directory:
+Copy the 2.2.0-7 inventory file into the 2.3-1.2 bundle directory:
 ----
-$ cd ansible-automation-platform-setup-bundle-2.1.2-1
-$ cp inventory ../ansible-automation-platform-setup-bundle-2.2.0-7/
+$ cd ansible-automation-platform-setup-bundle-2.2.0-7
+$ cp inventory ../ansible-automation-platform-setup-bundle-2.3-1.2/
```
Comment (ianf77) [inline downstream/modules/platform/proc-installing-the-ansible-builder-rpm.adoc:221]: should there be a difference between the formats for 2.2 and 2.3? it looks like 2.3-x.y and 2.2.x.y

Comment (emurtoug) [reply downstream/modules/platform/proc-installing-the-ansible-builder-rpm.adoc:221]: Hi Ian! I noticed this difference, too, but these are the version numbers provided to me by the reporter.

Comment (ianf77) [reply downstream/modules/platform/proc-installing-the-ansible-builder-rpm.adoc:221]: OK. Do you want it merging?

Comment (emurtoug) [reply downstream/modules/platform/proc-installing-the-ansible-builder-rpm.adoc:221]: Hi Ian, yes please! Thank you!

**Pull request merged**